go-controller/pkg/libovsdbops/model_client.go

```diff
@@ -200,19 +200,19 @@ e) if none of the above are true, ErrNotFound is returned.
 If BulkOp is set, update or mutate can happen accross multiple models found.
 */
 func (m *modelClient) CreateOrUpdate(opModels ...operationModel) ([]ovsdb.OperationResult, error) {
-	created, ops, err := m.createOrUpdateOps(nil, opModels...)
+	created, ops, err := m.createOrUpdateOps(nil, true, opModels...)
 	if err != nil {
 		return nil, err
 	}
 	return TransactAndCheckAndSetUUIDs(m.client, created, ops)
 }
 
 func (m *modelClient) CreateOrUpdateOps(ops []ovsdb.Operation, opModels ...operationModel) ([]ovsdb.Operation, error) {
-	_, ops, err := m.createOrUpdateOps(ops, opModels...)
+	_, ops, err := m.createOrUpdateOps(ops, true, opModels...)
 	return ops, err
 }
 
-func (m *modelClient) createOrUpdateOps(ops []ovsdb.Operation, opModels ...operationModel) (interface{}, []ovsdb.Operation, error) {
+func (m *modelClient) createOrUpdateOps(ops []ovsdb.Operation, doCreate bool, opModels ...operationModel) (interface{}, []ovsdb.Operation, error) {
```

Comment:
Does `doCreate` define if we want to actually create it or not?

Reply:
@JacobTanenbaum whether creation is allowed or not since the function is now called from both CreateOrUpdate() and just plain Update(). I think I'll change to `allowCreate`, what do you think?

Reply:
that sounds good to me